packages/terra-application-links/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 -----------------
+###Changed
```

Comment:
bump

Reply:
Keyboard navigation is present in the current release.